api/app.py

```diff
@@ -30,3 +32,29 @@ def coords_test():
 
     res = requests.get(req)
     return res.json()
+
+
+@app.route('/getCoordsOfAllStations')
+def getCoordsOfAllStations():
+    server = 'https://api.tidesandcurrents.noaa.gov/api/prod/datagetter'
+    date = 'latest'
+    # station = '8727520'  # Cedar Key
+    product = 'water_level'
+    datum = 'STND'
+    units = 'english'
+    time_zone = 'gmt'
+    application = 'uf_tides'
+    res_format = 'json'
+
+    stationIds = open("Database/station_ids.txt", "r")
+    resDictList = []
+    
+    for x in stationIds:
+        print(x)
+        station = x
+        req = server + '?' + '&'.join(['date=' + date, 'station=' + station, 'product=' + product, 'datum=' + datum,
```

Comment:
Probably no need to do a join here, I would just use the request string and format in the station variable.

Reply:
I'm a little confused what you mean by this one, are you suggesting doing something like this:
        `req = server + '?' + 'station=' + station`
where only the server is apart of the request message?

Reply:
It's all good, it works for now! I'll merge it.

Reply:
Sounds good, I tried to do what i typed above and it didn't seem to work, not sure if the api request requires the whole string (date, product, etc) to return data